apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -101,7 +102,8 @@ function ListBoxInline({ options, layout }) {
   const { translator, keyboardNavigation, themeApi, constraints } = useContext(InstanceContext);
 
   const { checkboxes = checkboxesOption } = layout || {};
-  const styles = getStyles({ app, themeApi, theme, components, checkboxes });
+  const themeSelectionColorsEnabled = flags.isEnabled('PS_22149_THEME_SELECTION_COLORS');
```

Comment:
potentially we can get the flag value earlier, and just pass that. Potentially put flags on some kine of context?

Reply:
looking at how we done it before.
then we have had the use of flags in https://github.com/qlik-oss/sn-list-objects
and a parameter in nebula.js

but I am not sure why we did it that way

Reply:
ok, I might have a look at that and revisit this. Would like to refactor a bit to align the code for selection colors in filterpane and in here